modules/file-integrity-operator-defining-custom-config.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-configuring.adoc
+
+[id="file-integrity-operator-defining-custom-config_{context}"]
+= Defining a custom File Integrity Operator configuration
+
+This example focuses on defining a custom configuration for a scanner that runs
+on the master nodes based on the default configuration provided for the
+`worker-fileintegrity` CR. This workflow might be useful if you are planning
+to deploy a custom software running as a daemon set and storing its data under
+`/opt/mydaemon` on the master nodes.
+
+.Procedure
+
+. Make a copy of the default configuration.
+
+. Edit the default configuration with the files that must be watched or excluded.
+
+. Store the edited contents in a new config map.
+
+. Point the `FileIntegrity` object to the new config map through the attributes in
+`spec.config`.
+
+. Extract the default configuration:
++
+[source,terminal]
+----
+$ oc extract cm/worker-fileintegrity --keys=aide.conf
+----
++
+This creates a file named `aide.conf` that you can edit. To illustrate how the
+Operator post-processes the paths, this example adds an exclude directory
+without the prefix:
++
+[source,terminal]
+----
+$ vim aide.conf
+----
++
+.Example output
+[source,json]
+----
+/hostroot/etc/kubernetes/static-pod-resources
+!/hostroot/etc/kubernetes/aide.*
+!/hostroot/etc/kubernetes/manifests
+!/hostroot/etc/docker/certs.d
+!/hostroot/etc/selinux/targeted
+!/hostroot/etc/openvswitch/conf.db
+----
++
+Exclude a path specific to master nodes:
++
+[source,json]
+----
+!/opt/mydaemon/
+----
++
+Catch everything else in `/etc`:
```

Comment:
@JAORMX I'm trying to think of a better way to say "Catch everything else" so that it translates better. Can you please clarify what you mean here? Thanks!

Reply:
I'm going with "Store the other content in" for now.